docs/scripts/reportBrokenLinks.js

```diff
@@ -36,26 +35,28 @@ parseDocFolder(
 );
 
 function getPageUrlFromLink(link) {
```

Comment:
Maybe renaming the function to stay align with the meaning
```suggestion
function removeApiLinkHash(link) {
```